### DISCUSSION TITLE
Translation are not working...

### DISCUSSION THREAD

**Discussion by marcobra** [pawelsalawa/sqlitestudio#4429] [General]
_###_ Details
Translation lang are not working

### Steps to reproduce
Change lang and restart sqlite studio lang remain the same English (tested with espanol de espana and italiano)

### Operating system
linux

### SQLiteStudio version
3.3.3 but was already not working with previous releases

**Comment by pawelsalawa** [comment]
In version 3.3.3 the Spanish translation is very incomplete, therefore if you switch to Spanish, you will still see mostly English messages & labels.

Version 3.4.0 (in developement) the Spanish translation is 33% done, so it's better, yet still far from complete.

**Reply by marcobra** [reply]
But italian translation is not in incomplete state, i'm the translator and the translation don't work selecting it...
I just sent a new bug issue.
Please verify by yourself  before to resend issue to discussion or please tell something more to dress this issue
Thanks
